social-share.android.js

```diff
@@ -36,8 +37,14 @@ module.exports = {
 		fos.flush();
 		fos.close();
 
-		intent.putExtra(android.content.Intent.EXTRA_STREAM,
-			android.net.Uri.fromFile(newFile));
+    var shareableFileUri;
+    var sdkVersionInt = parseInt(platform.device.sdkVersion);
+    if (sdkVersionInt >= 21) {
```

Comment:
Yes, 21 instead of 24. We can use the `FileProvider` on those lower API levels as well.